File: problemset/1-TwoSum/twosum.go
package twosum

func inArray(nums *[]int, target int, startindex int) int {
	for index := startindex; index < len(*nums); index++ {
		if (*nums)[index] == target {
			return index
		}
	}
	return 0
}

func twoSum(nums []int, target int) []int {
	for index, t := range nums {
		if index2 := inArray(&nums, target-t, index+1); index2 != 0 {
			return []int{index, index2}
		}
	}
	return []int{}
}

File: problemset/1-TwoSum/twosum.py
class Solution:
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        for index in range(len(nums)):
            t = nums[index]
            try:
                res = nums.index(target-t, index+1)
                return [index,res]
            except:
                pass
        